src/engine/source/api/src/test/handlers.cpp

```diff
@@ -621,8 +621,11 @@ api::Handler runPost(const shared_ptr<Router>& router)
         // Set optional parameters
         const auto debugMode = eRequest.has_debug_mode() ? eRequest.debug_mode() : eTest::DebugMode::OUTPUT_ONLY;
 
-        const auto protocolQueue =
-            eRequest.has_protocol_queue() ? eRequest.protocol_queue() : TEST_DEFAULT_PROTOCOL_QUEUE;
+        const uint8_t protocolQueue =
```

Comment:
Why is there a protocol queue limitation here? Please check this out

Reply:
It is due to the ascii table, the char can go from 0 to 9 that give their corresponding decimal of 48 and 57 respectively